osu.Game/Rulesets/Edit/DistancedHitObjectComposer.cs

```diff
@@ -125,6 +125,7 @@ protected override void Update()
             if (currentSnap > DistanceSpacingMultiplier.MinValue)
             {
                 currentDistanceSpacingButton.Enabled.Value = currentDistanceSpacingButton.Expanded.Value
+                                                             && !DistanceSpacingMultiplier.Disabled
```

Comment:
Notably, this also fixes a runtime crash that would occur if the button is clicked in catch editor. Which is good, but the description kinda looked like this was just a visual change at most.